andela/Healthchecks_Asgardians
# #145028379 User Configurable Report's Frequency

#### What does this PR do?
Add functionality to enable a user to select the frequency at which they would like to receive reports. The current setting allows only the option of getting monthly reports.
#### Description of Task to be completed?
The 'Reports' database to be user configurable to allow report frequency to be user selectable. The options should be daily, weekly and monthly.
#### How should this be manually tested?
- start server.
- navigate to 'Account Settings'
- Under 'Reports', click on checkbox to view implementation.
- Select a choice, and click save.
- Log out and back in to confirm that the previous  report frequency selected was perssited to the database.
#### What are the relevant pivotal tracker stories?
#145028379
#### Screenshots
<img width="865" alt="screen shot 2017-05-27 at 12 37 31" src="https://cloud.githubusercontent.com/assets/28535137/26520629/1bff9bfc-42df-11e7-9da9-36c1e047f301.png">
<img width="609" alt="screen shot 2017-05-27 at 13 20 45" src="https://cloud.githubusercontent.com/assets/28535137/26520650/6c6ef5a6-42df-11e7-9fb8-6ddbea743921.png">
<img width="615" alt="screen shot 2017-05-27 at 13 20 16" src="https://cloud.githubusercontent.com/assets/28535137/26520651/6c775ad4-42df-11e7-9f6d-ffb89d456908.png">
<img width="334" alt="screen shot 2017-05-27 at 13 20 04" src="https://cloud.githubusercontent.com/assets/28535137/26520652/6c7b5eea-42df-11e7-8f88-4d2e080c0ea5.png">








#### Questions:
